verl/workers/rollout/schemas.py

```diff
@@ -178,19 +248,66 @@ def update_metrics(self, metrics: Any, tool_id: str) -> None:
             self.metrics[tool_id] = []
         self.metrics[tool_id].append(metrics)
 
+    def _get_prompt_diffs(self, full_prompt: str, current_prompt: str, display_range: int = 10) -> List[Dict[str, Any]]:
+        s = difflib.SequenceMatcher(None, full_prompt, current_prompt, autojunk=False)
```

Comment:
Guess not all of our user familiar with current diff behavior, we need a little more comments or doc string